polaris.shopify.com/package.json

```diff
@@ -17,7 +17,7 @@
     "create-component": "generact --root src/components src/components/Template/Template.tsx",
     "gen-sitemap": "yarn next-sitemap",
     "get-props": "tsc ./scripts/get-props/src/get-props.ts --target esnext --module node16 --types \"react\" --outDir ./scripts/tmp && node ./scripts/tmp/scripts/get-props/src/get-props.js && rm -rf ./scripts/tmp",
-    "gen-assets": "node scripts/gen-assets.mjs && yarn get-props"
+    "gen-assets": "yarn get-props && node scripts/gen-assets.mjs"
```

Comment:
Have to run this first to ensure `.cache/props.json` exists, which is required for the site to build, which is required for the sitemap to be generated.

Reply:
I _think_ this also fixes a current prod issue where some URLs aren't showing up in the `sitemap.xml` (for example: `/components/actions/account-connection` does not exist in https://polaris.shopify.com/sitemap.xml)